src/Modules/ElementUtils/inheritance.js

```diff
@@ -6,13 +6,13 @@ const INHERITED_MEMBER_QUERY = {
   $or: [
     {
       'meta.links': {
-        href: 'http://refract.link/inherited-member/',
+        href: 'http://example.com/inherited-member/',
```

Comment:
Thing is - looking `example.com` in a code base gives me the idea of something unpolished-unfinished...but your mileage may vary